dashboard/config/scripts_json/csp9-2021.script_json

```diff
@@ -274,7 +274,7 @@
       "absolute_position": 9,
       "lockable": true,
       "has_lesson_plan": true,
-      "relative_position": 1,
+      "relative_position": 9,
```

Comment:
out of curiousity, how did you generate the list of lessons to modify the positions of?

Reply:
When I set has_lesson_plan for everything it was set based on lockable.  So if lockable was true then has_lesson_plan was false. I then updated the lessons in CSP 21-22 that we know are lockable but have lesson plans to set has_lesson_plan: true. So these are the lessons that would be impacted by this update to how we decided which URL because any other condition stays with the same url as before.  There are lessons in CSP 20-21 that also need this special update but because they are live in use we need to do a little more work to make that change.

Reply:
Thanks!